pkg/packagekit/updates.jsx

```diff
@@ -445,32 +398,23 @@ class ApplyUpdates extends React.Component {
     }
 
     render() {
-        var actionHTML, logRows;
-
-        if (this.state.actions.length > 0) {
-            const lastAction = this.state.actions[this.state.actions.length - 1];
-            actionHTML = (
-                <>
-                    <strong>{ PK_STATUS_STRINGS[lastAction.status] || PK_STATUS_STRINGS[PK.Enum.STATUS_UPDATE] }</strong>
-                    &nbsp;{lastAction.package}
-                </>);
-            logRows = this.state.actions.slice(0, -1).map((action, i) => (
-                <tr key={action.package + i}>
-                    <th>{PK_STATUS_LOG_STRINGS[action.status] || PK_STATUS_LOG_STRINGS[PK.Enum.STATUS_UPDATE]}</th>
-                    <td>{action.package}</td>
-                </tr>));
-        } else {
-            actionHTML = _("Initializing...");
+        if (this.state.actions.length === 0) {
+            return <EmptyStatePanel title={ _("Initializing...") }
+                                    secondary={ <Button variant="secondary" onClick={this.props.onCancel} isDisabled={!this.props.allowCancel}>{_("Cancel")}</Button> }
```

Comment:
This button is being used below again, so it could be factorized. But not a biggie.